libhb/common.h

```diff
@@ -501,14 +501,16 @@ struct hb_job_s
          cfr:               0 (vfr), 1 (cfr), 2 (pfr) [see render.c]
          pass:              0, 1 or 2 (or -1 for scan)
          areBframes:        boolean to note if b-frames are used */
-#define HB_VCODEC_MASK         0x00FFFFF
+#define HB_VCODEC_MASK         0x08FFFFF
```

Comment:
This doesn't seem to be necessary.  The mask already covered the bits you added.  Is there something non-obvious I'm missing?